Assets/Scripts/Rocket/RocketBodyCollider.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RocketBodyCollider : MonoBehaviour
+{
+    [SerializeField] private RocketStatusControl rocketStatusControl;
+    [SerializeField] private RocketDestruction rocketDestruction;
+
+    private void OnTriggerEnter2D(Collider2D collider)
+    {
+        if (collider.gameObject.tag != "Item" && rocketStatusControl.isDead == false)
```

Comment:
since `"Item"` is a constant value used in at least two places you could create a `private const string ItemTag = "Item"` in the class in case you need to rename it in the future.